### PR TITLE
Adds runner warnings to DataImport 5957

### LINKED ISSUE #5957: Change generic error when importing more than 10 files
Right now, if a user imports a compressed file with more than 10 datasets, the importer fails generically. It would be nice if we could add an error with a specific error, specially now that we are adding the functionality of creating multilayered maps from a zip file.

Relevant code:
https://github.com/CartoDB/cartodb/blob/master/services/importer/lib/importer/runner.rb#L27
https://github.com/CartoDB/cartodb/blob/master/services/importer/lib/importer/runner.rb#L249-L261

```ruby
@importer_stats.timing('import') do
  unpacker.source_files.each_with_index { |source_file, index|

    next if (index >= MAX_TABLES_PER_IMPORT)
    @job.new_table_name if (index > 0)

    log.store   # Checkpoint-save
    log.append "Filename: #{source_file.fullpath} Size (bytes): #{source_file.size}"
    import_stats = execute_import(source_file, @downloader)
    @stats << import_stats

  }
end
```

### PR DESCRIPTION
**Pull request by gfiorav**
Fixes #5957

When importing a compressed file with more files than admitted (10 right now), this warning pops up after import:

screen shot 2015-10-29 at 18 39 01

If user get the another warning for no enough `max_layers` but the zip uploaded had more than 10 files, he now gets a notification:

screen shot 2015-10-29 at 18 40 31

### CODE REVIEW

**Comment by gfiorav** [comment]
@Kartones could you CR the backend part please? 

@xavijam could you review the front end part or call someone to do it?

Thanks to both of you!

File: lib/assets/javascripts/cartodb/common/views/warnings_details_view.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ var cdb = require('cartodb.js');
 module.exports = cdb.core.View.extend({
   _TEMPLATES: {
     'partial_import': 'common/views/partial_import_details',
+    'too_many_files': 'common/views/too_many_files_details',
     default: 'common/views/error_details'
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/common/views/warnings_details_view.js]: Default option is not needed right?. Also, if there is only two possibilities, I'd create a ternary.
```js
var template = cdb.templates.getTemplate(this._TEMPLATES[ warnings.max_tables_per_import ? 'too_many_files' : 'partial_import' ]);
```

Comment (gfiorav) [inline lib/assets/javascripts/cartodb/common/views/warnings_details_view.js]: Yep, nice!

**Comment by gfiorav** [comment]
Retest this, please

File: config/app_config.yml.sample [hunk 2]
```diff
@@ -1,8 +1,4 @@
 defaults: &defaults
-  # If uncommented, most images like static map previews and twitter card image urls will use this CDN urls
```
Comment (Kartones) [inline config/app_config.yml.sample]: bad merge, you're removing config...

File: config/app_config.yml.sample [hunk 3]
```diff
@@ -271,11 +268,6 @@ defaults: &defaults
       app_key:              ''
       app_secret:           ''
       callback_url:         ''
-    box:
```
Comment (Kartones) [inline config/app_config.yml.sample]: Same here, please get back this lines

File: config/app_config.yml.sample [hunk 4]
```diff
@@ -14,6 +10,7 @@ defaults: &defaults
   subdomainless_urls: false
   http_port:           3000 # nil|integer. HTTP port to use when building urls. Leave empty to use default (80)
   https_port:          # nil|integer. HTTPS port to use when building urls. Leave empty to use default (443)
+  maps_api_cdn_template: # nil|string. E.g. '{protocol}://{zone}.cartocdn/{user}/api/v1/map/static/named/{tpl_id}/{width}/{height}.png'
```
Comment (Kartones) [inline config/app_config.yml.sample]: Also bad, this one is actually deleted

File: config/app_config.yml.testing [hunk 5]
```diff
@@ -225,11 +225,6 @@ defaults: &defaults
       app_key:  ''
       app_secret: ''
       callback_url: ''
-    box:
```
Comment (Kartones) [inline config/app_config.yml.testing]: Also should be here..

File: config/app_config.yml.sample [hunk 6]
```diff
@@ -311,6 +303,13 @@ defaults: &defaults
             ratelimit_concurrency:  3
             ratelimit_ttl:          4
             ratelimit_wait_secs:    0.1
+  feature_flags:
```
Comment (Kartones) [inline config/app_config.yml.sample]: uhmm and are you sure we have here feature flags? in the config? /cc @juanignaciosl

File: app/controllers/carto/api/data_import_presenter.rb [hunk 7]
```diff
@@ -181,6 +181,8 @@ def get_warnings
           warnings.merge!(user_max_layers: @data_import.user.max_layers)
         end
 
+        warnings.merge!(JSON.parse(@data_import.runner_warnings)) if !@data_import.runner_warnings.nil?
```
Comment (Kartones) [inline app/controllers/carto/api/data_import_presenter.rb]: I would add a small test here (of `DataImportPresenter`) so you have a quick end-to-end test that warnings get propagated from the import to data import model and then read here properly... but your call

**Comment by Kartones** [comment]
Config has bad merges, but apart from that... looks fine so once those are fixed :+1:

**Comment by gfiorav** [comment]
@Kartones Don't know what went wrong rebasing... Sorry for that. I'll fix the merge. Thanks for the CR!

File: app/controllers/carto/api/data_import_presenter.rb (hunk 7 above)
Comment (gfiorav) [inline app/controllers/carto/api/data_import_presenter.rb]: Added!

Comment (Kartones) [inline app/controllers/carto/api/data_import_presenter.rb]: :+1:

**Comment by juanignaciosl** [comment]
I've received a notification about a comment but I think it's no longer here, let me know if I can help.

**Comment by gfiorav** [comment]
Thanks @juanignaciosl , it was a mistake rebasing. Nothing to see here :smile:

**Pull request merged**